src/public/meeting.ts

```diff
@@ -159,6 +159,20 @@ export namespace meeting {
     sendMessageToParent('toggleIncomingClientAudio', callback);
   }
 
+  /**
+   * Allows an app to mute the microphone in the meeting.
+   * @param callback Callback contains 2 parameters, error and result.
+   * error can either contain an error of type SdkError, incase of an error, or true when mute is successful
```

Comment:
* error can either contain an error of type SdkError, incase of an error, or null when mute is successful
   * result can either contain the true value, incase of a successful mute or null when the mute fails
   * result: True means microphone is muted